src/lib/legacy.ts

```diff
@@ -6712,15 +6728,16 @@ const toZigbee2 = {
         convertSet: async (entity, key, value: any, meta) => {
             if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
                 await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value * 10);
-            } else {
-                if (value <= 5) value = Math.round(value*10);
-                await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value);
             }
+            await sendDataPointValue(entity, dataPoints.moesHeatingSetpoint, value);
         },
     } as Tz.Converter,
     moes_thermostat_deadzone_temperature: {
         key: ['deadzone_temperature'],
-        convertSet: async (entity, key, value, meta) => {
+        convertSet: async (entity, key, value: any, meta) => {
+            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
+                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value * 10);
+            }
             await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value);
```

Comment:
Make this an if else, otherwise you send the value twice, e.g.

```js
            if (['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value * 10);
            } else {
                await sendDataPointValue(entity, dataPoints.moesDeadZoneTemp, value);
            }
```